docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md

```diff
@@ -96,7 +96,9 @@ global:
 For authentication, the Camunda components use the scopes `email`, `openid`, and `offline_access`.
 
 :::note
-Web Modeler does not yet support authentication with a generic OIDC provider.
+Web Modeler and the Connectors do not yet support authentication with a generic OIDC provider.
+
+When using OIDC, please disable Connectors in your deployment (see https://docs.camunda.io/docs/next/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider/).
```

Comment:
```suggestion
When using OIDC, please disable Connectors in your deployment (see https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/values.yaml#L2572).
```

Reply:
Or https://docs.camunda.io/docs/next/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd/?

Reply:
Regardless, let's make sure to use `[anchor text like this for links]()`.

Reply:
Let's use https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/values.yaml#L2572 please.

@christinaausley can you please change the PR? (I am not at my laptop anymore)

Reply:
Please get some sleep @MaxTru 😄 I'll get this adjusted.